[SMA] Compilation error in Win

**Description**
I've just updated the master and it does not compile because of the StructuralMechApp.

Error message:

D:\KRATOS_MULTIPHYSICS\Kratos\applications\StructuralMechanicsApplication\custom_response_functions\adjoint_e
lements\adjoint_finite_difference_truss_element_3D2N.cpp(91): error C3861: **'_Pragma**': no se encontró el ident
ificador [D:\KRATOS_MULTIPHYSICS\Kratos\build\Release\applications\StructuralMechanicsApplication\KratosStruc
turalMechanicsCore.vcxproj]

D:\KRATOS_MULTIPHYSICS\Kratos\applications\StructuralMechanicsApplication\custom_elements\z_strain_driven_2p5
_small_displacement.cpp(164): error C3861: '_Pragma': no se encontró el identificador [D:\KRATOS_MULTIPHYSICS
\Kratos\build\Release\applications\StructuralMechanicsApplication\KratosStructuralMechanicsCore.vcxproj]

D:\KRATOS_MULTIPHYSICS\Kratos\applications\StructuralMechanicsApplication\custom_elements\truss_element_3D2N.
cpp(453): error C3861: '_Pragma': no se encontró el identificador [D:\KRATOS_MULTIPHYSICS\Kratos\build\Releas
e\applications\StructuralMechanicsApplication\KratosStructuralMechanicsCore.vcxproj]

D:\KRATOS_MULTIPHYSICS\Kratos\applications\StructuralMechanicsApplication\custom_elements\solid_shell_element
_sprism_3D6N.cpp(1365): error C3861: '_Pragma': no se encontró el identificador [D:\KRATOS_MULTIPHYSICS\Krato
s\build\Release\applications\StructuralMechanicsApplication\KratosStructuralMechanicsCore.vcxproj]

And in some other places... @KratosMultiphysics/structural-mechanics

--- a/kratos/includes/checks.h
+++ b/kratos/includes/checks.h
@@ -171,7 +171,7 @@ try {                                                                           
 // this macro is to be removed, as it is no longer required to check the keys of Variables (are now assigned at compiletime)
 #if defined(_MSC_VER)
 #define KRATOS_CHECK_VARIABLE_KEY(TheVariable) \
-    __pragma(message("\"'KRATOS_CHECK_VARIABLE_KEY' macro is no longer needed and can be safely removed\"")); \
+    __pragma(message("\"'KRATOS_CHECK_VARIABLE_KEY' macro is no longer needed and can be safely removed\""));
 #else
 #define KRATOS_CHECK_VARIABLE_KEY(TheVariable) \
     _Pragma ("message( \"'KRATOS_CHECK_VARIABLE_KEY' macro is no longer needed and can be safely removed\")"); \

--- a/kratos/includes/checks.h
+++ b/kratos/includes/checks.h
@@ -169,13 +169,14 @@ try {                                                                           
 }
 
 // this macro is to be removed, as it is no longer required to check the keys of Variables (are now assigned at compiletime)
-#define KRATOS_CHECK_VARIABLE_KEY(TheVariable) \
 #if defined(_MSC_VER)
+#define KRATOS_CHECK_VARIABLE_KEY(TheVariable) \
     __pragma(message("\"'KRATOS_CHECK_VARIABLE_KEY' macro is no longer needed and can be safely removed\"")); \
 #else
+#define KRATOS_CHECK_VARIABLE_KEY(TheVariable) \
     _Pragma ("message( \"'KRATOS_CHECK_VARIABLE_KEY' macro is no longer needed and can be safely removed\")"); \
-#endif
     TheVariable.Key(); // adding dummy usage to avoid unused variable warnings
+#endif
 
 #define KRATOS_CHECK_VARIABLE_IN_NODAL_DATA(TheVariable, TheNode)                          \
     KRATOS_ERROR_IF_NOT(TheNode.SolutionStepsDataHas(TheVariable))                         \

--- a/kratos/includes/checks.h
+++ b/kratos/includes/checks.h
@@ -170,7 +170,11 @@ try {                                                                           
 
 // this macro is to be removed, as it is no longer required to check the keys of Variables (are now assigned at compiletime)
 #define KRATOS_CHECK_VARIABLE_KEY(TheVariable) \
+#if defined(_MSC_VER)
+    __pragma(message("\"'KRATOS_CHECK_VARIABLE_KEY' macro is no longer needed and can be safely removed\"")); \
+#else
     _Pragma ("message( \"'KRATOS_CHECK_VARIABLE_KEY' macro is no longer needed and can be safely removed\")"); \
+#endif
     TheVariable.Key(); // adding dummy usage to avoid unused variable warnings
 
 #define KRATOS_CHECK_VARIABLE_IN_NODAL_DATA(TheVariable, TheNode)                          \

